2200033_박지환/반반_D3.py

```diff
@@ -0,0 +1,20 @@
+T = int(input())
+
+for i in range(1, T+1):
+    word = input()
+    answer_list = []
+    cnt = 0
+
+    for j in word:
+        answer_list.append(j)
```

Comment:
```suggestion
    answer_list = list(input())
    cnt = 0
```
위와 같이 작성하셔도 괜찮을 것 같습니다 😄